### PR TITLE
init mysql

### PR DESCRIPTION
**Pull request by tuuna**

### CODE REVIEW

File: Week1-Database-Introduction/MySQLExperiment.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+# MySQL 入门指南
+
+[TOC]
```
Comment (PiotrNewt) [inline Week1-Database-Introduction/MySQLExperiment.md:3]: [TOC] for what?

File: Week1-Database-Introduction/MySQLExperiment.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+•**查询缓存:** 执行查询语句的时候，会先查询缓存（MySQL 8.0 版本后移除，因为这个功能不太实用）。
+
+•**分析器:** 没有命中缓存的话，SQL 语句就会经过分析器，分析器说白了就是要先看你的 SQL 语句要干嘛，再检查你的 SQL 语句语法是否正确。
+
+•**优化器：** 按照 MySQL 认为最优的方案去执行。
+
+•**执行器:** 执行语句，然后从存储引擎返回数据。
+
+![sql_structure](/Users/laijie/Git/leetcode_sign/image/sql_structure.png)
```
Comment (PiotrNewt) [inline Week1-Database-Introduction/MySQLExperiment.md:77]: image link is broken

File: Week1-Database-Introduction/MySQLExperiment.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
+# MySQL 入门指南
```
Comment (PiotrNewt) [inline Week1-Database-Introduction/MySQLExperiment.md:1]: Please rewrite the format for full doc, very messy

File: Week1-Database-Introduction/MySQLExperiment.md (hunk 1 above)
Comment (tuuna) [reply Week1-Database-Introduction/MySQLExperiment.md:3]: 目录啊

Comment (PiotrNewt) [reply Week1-Database-Introduction/MySQLExperiment.md:3]: 似乎没有生成目录？可以 preview 一下

**Pull request merged**